manifests/Makefile

```diff
@@ -0,0 +1,38 @@
+ARGO_ROLLOUTS_VERSION := v0.10.2
+
+TEMP_DIR := $(shell mktemp -d)
+
+.PHONY: argo-rollouts
+argo-rollouts: argo-rollouts-crds argo-rollouts-base
+
+.PHONY: argo-rollouts-crds
+argo-rollouts-crds:
```

Comment:
Maybe we should make this more generic e.g. `make upstreams` that looks at each file in the upstream directory and then renames it to `kustomization.yaml` and places the result in the file with the name ?

```bash
Makefile
upstream/
    crd/
        argo-rollouts.yaml
    argo-rollououts.yaml
```

Reply:
Yeah, I like that. So:
1. Create a new directory `manifests/upstream`
2. Move `manifests/crd/argo-rollouts.yaml` to `manifests/upstream/crd`
3. Move `manifest/argo-rollouts.yaml` to `manifests/upstream`
4. Modify `Makefile` and make it generic by creating a new task that iterates over files in `upstream` then for each file: reads the file name then runs the task that has the same name with the file name (without the file extension)
5. Other tasks remain the same

Sounds right? I'm not sure about this part
> then renames it to kustomization.yaml and places the result in the file with the name 

We will apply this pattern for `argo-rollouts` first and going forward we will move other production runtimes manifests to follow this yaml, right?.

Reply:
5. So it would go
`mv manifests/upstream/argo-rollouts.yaml $TMP/kustomization.yaml`
`kustomize build $TMP/ > manifests/argo-rollouts.yaml`

Reply:
And yes we would apply this pattern going forward, and then also backport to existing runtimes.

Reply:
This is done. The `Makefile` is now much more elegant (I think 😄 ). It will rebuild manifests only if the corresponding upstream file is updated from the last time the manifest was built.